From fa9c14b24df673cffad319351ce48dd1d64193f3 Mon Sep 17 00:00:00 2001
From: BastianZim <10774221+BastianZim@users.noreply.github.com>
Date: Fri, 1 Oct 2021 14:32:59 +0200
Subject: [PATCH] Patch for downstream

---
 Cbc/cbc.pc.in | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/Cbc/cbc.pc.in b/Cbc/cbc.pc.in
index 4d78b812..be3f8670 100644
--- a/Cbc/cbc.pc.in
+++ b/Cbc/cbc.pc.in
@@ -7,6 +7,7 @@ Name: CBC
 Description: COIN-OR Branch and Cut
 URL: https://projects.coin-or.org/Cbc
 Version: @PACKAGE_VERSION@
-Libs: -L${libdir} -lCbcSolver -lCbc @CBCLIB_PCLIBS@
+Libs: -L${libdir} -lCbcSolver -lCbc
+Libs.private: @CBCLIB_PCLIBS@
 Cflags: -I${includedir}
-Requires: @CBCLIB_PCREQUIRES@
+Requires.private: @CBCLIB_PCREQUIRES@
-- 
2.35.1.windows.2

