From baa936318b45548bace628c9e20df4918023e8e5 Mon Sep 17 00:00:00 2001
From: BastianZim <10774221+BastianZim@users.noreply.github.com>
Date: Wed, 3 Nov 2021 18:24:21 +0100
Subject: [PATCH 2/2] Patch build

---
 Cbc/configure | 28 ++++++++++++++--------------
 1 file changed, 14 insertions(+), 14 deletions(-)

diff --git a/Cbc/configure b/Cbc/configure
index 5bcb83b2..8208b550 100755
--- a/Cbc/configure
+++ b/Cbc/configure
@@ -21475,9 +21475,9 @@ if test $coin_has_clp = notGiven; then
 
 
 if test -n "$PKG_CONFIG" ; then
-  if $PKG_CONFIG --exists "osi-clp"; then
-    CLP_VERSIONS=`$PKG_CONFIG --modversion "osi-clp" 2>/dev/null | tr '\n' ' '`
-     cflags=`$PKG_CONFIG --cflags "osi-clp" 2>/dev/null`
+  if $PKG_CONFIG --exists "clp osi-clp"; then
+    CLP_VERSIONS=`$PKG_CONFIG --modversion "clp osi-clp" 2>/dev/null | tr '\n' ' '`
+     cflags=`$PKG_CONFIG --cflags "clp osi-clp" 2>/dev/null`
     # pkg-config cannot handle spaces, so CYGPATH_W cannot be put into .pc files
 	# thus, we modify the cflags extracted from pkg-config by putting CYGPATH_W behind -I's
 	# but only do this if is not trivial
@@ -21486,8 +21486,8 @@ if test -n "$PKG_CONFIG" ; then
       cflags=`echo $cflags | sed -e 's/-I\([^ ]*\)/-I\`${CYGPATH_W} \1\`/g'`
     fi
     CLP_CFLAGS="$cflags"
-    CLP_LIBS=`$PKG_CONFIG --libs "osi-clp" 2>/dev/null`
-    CLP_DATA=`$PKG_CONFIG --variable=datadir "osi-clp" 2>/dev/null`
+    CLP_LIBS=`$PKG_CONFIG --libs "clp osi-clp" 2>/dev/null`
+    CLP_DATA=`$PKG_CONFIG --variable=datadir "clp osi-clp" 2>/dev/null`
      coin_has_clp=yes
         echo "$as_me:$LINENO: result: yes: $CLP_VERSIONS" >&5
 echo "${ECHO_T}yes: $CLP_VERSIONS" >&6
@@ -21498,22 +21498,22 @@ echo "${ECHO_T}yes: $CLP_VERSIONS" >&6
         then
           CLP_LIBS=`echo " $CLP_LIBS " | sed -e 's/ \(\/[^ ]*\/\)\([^ ]*\)\.lib / \`$(CYGPATH_W) \1 | sed -e "s|\\\\\\\\\\\\\\\\\\\\|\/|g"\`\2.lib /g'`
         fi
-        CLP_PCREQUIRES="osi-clp"
+        CLP_PCREQUIRES="clp osi-clp"
 
         # augment X_PCREQUIRES, X_CFLAGS, and X_LIBS for each build target X in CbcLib CbcGeneric
 
-          CBCLIB_PCREQUIRES="osi-clp $CBCLIB_PCREQUIRES"
+          CBCLIB_PCREQUIRES="clp osi-clp $CBCLIB_PCREQUIRES"
           CBCLIB_CFLAGS="$CLP_CFLAGS $CBCLIB_CFLAGS"
           CBCLIB_LIBS="$CLP_LIBS $CBCLIB_LIBS"
 
-          CBCGENERIC_PCREQUIRES="osi-clp $CBCGENERIC_PCREQUIRES"
+          CBCGENERIC_PCREQUIRES="clp osi-clp $CBCGENERIC_PCREQUIRES"
           CBCGENERIC_CFLAGS="$CLP_CFLAGS $CBCGENERIC_CFLAGS"
           CBCGENERIC_LIBS="$CLP_LIBS $CBCGENERIC_LIBS"
 
 
 
   else
-    CLP_PKG_ERRORS=`$PKG_CONFIG $pkg_short_errors --errors-to-stdout --print-errors "osi-clp"`
+    CLP_PKG_ERRORS=`$PKG_CONFIG $pkg_short_errors --errors-to-stdout --print-errors "clp osi-clp"`
       coin_has_clp=notGiven
         echo "$as_me:$LINENO: result: not given: $CLP_PKG_ERRORS" >&5
 echo "${ECHO_T}not given: $CLP_PKG_ERRORS" >&6
@@ -21549,8 +21549,8 @@ CLP_DATA_INSTALLED=
 CLP_PCLIBS=
 CLP_PCREQUIRES=
 
-# initial list of dependencies is "osi-clp", but we need to filter out version number specifications (= x, <= x, >= x, != x)
-projtoprocess="osi-clp"
+# initial list of dependencies is "clp osi-clp", but we need to filter out version number specifications (= x, <= x, >= x, != x)
+projtoprocess="clp osi-clp"
 
 # we first expand the list of projects to process by adding all dependencies just behind the project which depends on it
 # further, we collect the list of corresponding .pc files, but do this in reverse order, because we need this order afterwards
@@ -21751,15 +21751,15 @@ _ACEOF
     CLP_LIBS_INSTALLED=`echo " $CLP_LIBS_INSTALLED" | sed -e 's/ \(\/[^ ]*\/\)/ \`$(CYGPATH_W) \1\`/g' -e 's/ -l\([^ ]*\)/ lib\1.lib/g' -e 's/ -L\([^ ]*\)/ -libpath:\`$(CYGPATH_W) \1\`/g'`
   fi
 
-  CLP_PCREQUIRES="osi-clp"
+  CLP_PCREQUIRES="clp osi-clp"
 
-    CBCLIB_PCREQUIRES="osi-clp $CBCLIB_PCREQUIRES"
+    CBCLIB_PCREQUIRES="clp osi-clp $CBCLIB_PCREQUIRES"
     CBCLIB_CFLAGS="$CLP_CFLAGS $CBCLIB_CFLAGS"
     CBCLIB_LIBS="$CLP_LIBS $CBCLIB_LIBS"
     CBCLIB_CFLAGS_INSTALLED="$CLP_CFLAGS_INSTALLED $CBCLIB_CFLAGS_INSTALLED"
     CBCLIB_LIBS_INSTALLED="$CLP_LIBS_INSTALLED $CBCLIB_LIBS_INSTALLED"
 
-    CBCGENERIC_PCREQUIRES="osi-clp $CBCGENERIC_PCREQUIRES"
+    CBCGENERIC_PCREQUIRES="clp osi-clp $CBCGENERIC_PCREQUIRES"
     CBCGENERIC_CFLAGS="$CLP_CFLAGS $CBCGENERIC_CFLAGS"
     CBCGENERIC_LIBS="$CLP_LIBS $CBCGENERIC_LIBS"
     CBCGENERIC_CFLAGS_INSTALLED="$CLP_CFLAGS_INSTALLED $CBCGENERIC_CFLAGS_INSTALLED"
-- 
2.35.1.windows.2

